sdk/data/aztables/cloud_config.go

```diff
@@ -12,12 +12,12 @@ const ServiceName cloud.ServiceName = "data/aztables"
 
 func init() {
```

Comment:
We should document that the default is for table storage which is `https://storage.azure.com`. We only consume these values if it's a Cosmos endpoint.

Reply:
```go
// for table storage endpoints, these values are ignored as the audience is always https://storage.azure.com
// for cosmos endpoints, we will use the audience as specified here
```